src/features/registry.js

```diff
@@ -12,6 +12,8 @@ systemJSPrototype.get = function (id) {
 };
 
 systemJSPrototype.set = function (id, module) {
+  if (this[REGISTRY][id]) return false;
```

Comment:
Is this a breaking change? You can no longer call `System.set` on a module that already exists?

Reply:
We could potentially allow set to be called for modules that are loaded or errored.... the main thing I was trying to lock down here was setting modules while in a loading state.

The behaviour for a set in the loaded states would effectively be the same as `delete` followed by `set`. Will add that.

Reply:
I've updated this.